regexes/oss.yml

```diff
@@ -192,6 +192,20 @@
   name: 'FydeOS'
   version: ''
 
+##########
+# Revenge OS
+##########
+- regex: 'revengeos_x2'
```

Comment:
Does that mean that there is only one device where this OS will be detected? Aren't there others or is it simply not detectable on other devices?

Reply:
Most likely there are no identifiers for the device in this OS
revengeos_x2 can be mapped to package name 
https://sourceforge.net/projects/revengeos/files/x2/